integration-test/src/main/resources/application.yml

```diff
@@ -36,6 +36,9 @@ integrationtest:
   redbeams:
     server: http://localhost
   user:
+    realUms:
+      accountKey: default
+      environmentKey: dev
```

Comment:
these are not user related informations, i would move them at least one level higher, also these structure would be simpler:
```
ums:
  account: default
  environment: dev
```
or something like that